ibis/backends/clickhouse/compiler/values.py

```diff
@@ -597,7 +592,8 @@ def _in_column(op, *, value, options, **_):
     return value.isin(options.this if isinstance(options, sg.exp.Subquery) else options)
 
 
-_NUM_WEEKDAYS = 7
+_DAYS = calendar.day_name
+_NUM_WEEKDAYS = len(_DAYS)
```

Comment:
Could we use just `7` everywhere? Currently I have to track 3 variables whereas it is always just `7`.

```py
weekdays = _NUM_WEEKDAYS
_NUM_WEEKDAYS = len(_DAYS)
_DAYS = calendar.day_name
```

Reply:
I'd prefer to have `_NUM_WEEKDAYS` removed entirely, can be done in a follow-up.